content/uxcs/basics.mdx

```diff
@@ -18,6 +18,23 @@ Zendesk doesn’t speak jargon. Use Plain Language instead. Jargon is confusing,
 and can feel downright icky. It can also result in non-inclusive language, which goes against
 our values. Replace jargon with plain language and define acronyms on first use.
 
+We now support a _wide_ variety of **styles** and `inline code` syntaxes.
+
+1. Ordered lists
+1. Are now
+1. Supported
+
+> Including long-form block quotes
+
+- As well as unordered
+- Lists
```

Comment:
@talia-eisenberg please note that Garden has the ability to override the bullet style for unordered lists. (See https://garden.zendesk.com/react-components/typography/#lists.) We fallback to the browser default, but we can specify it to be `disc` which is probably the more typical treatment.